ingress/ingress-https-setup.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+# path to dashboard repo
+export REPO_TOP_LEVEL=$(git rev-parse --show-toplevel)
+# path to cert & keys
+export CERTIFICATE_DIR="/etc/ssl/certs/tekton-dashboard"
+
+# certificate data
+# *Make sure names contain only lowercase alphanumeric characters, . or -. Must start & end with alphanumeric characters*
+export CERTIFICATE_KEY=""
+export CERTIFICATE_KEY_PASSPHRASE=""
+export CERTIFICATE_NAME=""
+export CERTIFICATE_SECRET_NAME=""
+export IP_ADDRESS=$(ip route get 8.8.8.8 | awk '{print $NF; exit}')
```

Comment:
The IP address must be one of host within the kubernetes cluster.  Should this be executed one of the cluster hosts?

Reply:
I believe it should be executed in the cluster the hosts wants to access in the browser

Reply:
OK.  Would you put the assumption in the README.md?